src/mad_spark_alt/core/simple_qadi_orchestrator.py

```
@@ -381,13 +392,17 @@ async def _run_deduction_phase(
-                    r"ANSWER:\s*(.+?)(?=Action Plan:|$)", content, re.DOTALL
+                    rf"{ANSWER_PREFIX}\s*(.+?)(?={ACTION_PLAN_PREFIX}|$)",
+                    content,
+                    re.DOTALL,
                 )
                 answer = answer_match.group(1).strip() if answer_match else ""
 
                 # Extract action plan
                 action_plan = []
-                plan_match = re.search(r"Action Plan:\s*(.+?)$", content, re.DOTALL)
+                plan_match = re.search(
+                    rf"{ACTION_PLAN_PREFIX}\s*(.+?)$", content, re.DOTALL
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)